<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>species_params</class>
 <widget class="QWidget" name="species_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>187</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">species_params</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
     <property name="buddy">
      <cstring>nameLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="presetLabel">
     <property name="text">
      <string>Preset</string>
     </property>
     <property name="buddy">
      <cstring>presetComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="presetComboBox">
     <item>
      <property name="text">
       <string>Choose...</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="massKgLabel">
     <property name="text">
      <string>Mass (kg)</string>
     </property>
     <property name="buddy">
      <cstring>massKgLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="epsilonJLabel">
     <property name="text">
      <string>Epsilon (J)</string>
     </property>
     <property name="buddy">
      <cstring>epsilonJLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="sigmaMLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sigmaMLabel">
     <property name="text">
      <string>Sigma (m)</string>
     </property>
     <property name="buddy">
      <cstring>sigmaMLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="massKgLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="epsilonJLineEdit"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>presetComboBox</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>massKgLineEdit</tabstop>
  <tabstop>sigmaMLineEdit</tabstop>
  <tabstop>epsilonJLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
